test/helper.js

```diff
@@ -0,0 +1,21 @@
+const greetbot = require('../src/index.js');
+const user_id = process.env.SLACK_USER_ID;
```

Comment:
maybe you could also do something like
```
const {
SLACK_USER_ID,
SLACK_DM_ID,
... } = process.env;
```
that way you dont need the variable declarations